src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx

```diff
@@ -126,20 +106,16 @@ const Room: React.FC<RoomProps> = ({
     setSeatedAtTable && setSeatedAtTable("");
   }, [firebase, profile, setSeatedAtTable, user, venueName]);
 
-  useEffect(() => {
-    if (!token) return;
-
-    let localRoom: Video.Room;
-
-    const participantConnected = (participant: Video.Participant) => {
-      setParticipants((prevParticipants) => [
-        // Hopefully prevents duplicate users in the participant list
-        ...prevParticipants.filter((p) => p.identity !== participant.identity),
-        participant,
-      ]);
-    };
+  const participantConnected = useCallback((participant: Video.Participant) => {
+    setParticipants((prevParticipants) => [
+      // Hopefully prevents duplicate users in the participant list
+      ...prevParticipants.filter((p) => p.identity !== participant.identity),
+      participant,
+    ]);
+  }, []);
 
-    const participantDisconnected = (participant: Video.Participant) => {
+  const participantDisconnected = useCallback(
+    (participant: Video.Participant) => {
       setParticipants((prevParticipants) => {
         if (!prevParticipants.find((p) => p === participant)) {
           // @debt Remove when root issue found and fixed
```

Comment:
And what is the issue? Any number or link as a reference?

Reply:
I'm not sure what this was referring to. It's legacy